docs/pages/includes/install-linux.mdx

```diff
@@ -1,4 +1,4 @@
-<ScopedBlock scope={["oss", "cloud"]}>
+<ScopedBlock scope={["oss"]}>
```

Comment:
With this change, there's no `ScopedBlock` for Cloud users, meaning that users with the `cloud` scope will see missing content in some pages. 

[This partial](https://github.com/gravitational/teleport/blob/2e32065ea65c1763aa8544c2f2b2aac8d9ca4349/docs/pages/includes/install-agent-cloud.mdx) includes instructions for downloading a version of Teleport matching the Cloud version, though we would need to change the wording so it doesn't say "Node" (maybe, "On the host where you will run `teleport`" instead?).

Reply:
TBH I'd rather we redirect the user to Cloud Downloads page (as is the case now) which has up-to-date download instructions for various installation methods. This partial only includes a single Linux x86_64 download which may be work for all scenarios.

Reply:
Would it be possible to change `install-linux.mdx` to include a set of instructions to follow for Cloud users? Currently, Cloud users see something like this screenshot from the `postgres-redshift.mdx` guide (empty space where installation instructions should be):

<img width="766" alt="image" src="https://user-images.githubusercontent.com/25250668/182954966-60eb8cc9-e4c5-45d4-871b-ff585710361a.png">

Ideally, in step-by-step tutorials like this, users shouldn't need to leave the page to see the information required to follow the guide.

Reply:
Hmm yeah that's not great. I think in this case your suggestion to use the partial we have makes sense. I'll update.

Reply:
I've update this partial now to include a section for cloud scope. It's basically the same as OSS but uses cloud versions and excludes legacy repositories.